air-quality-backend/src/etl/in_situ/openaq_adapter.py

```diff
@@ -63,6 +65,10 @@ def _create_document(measurement, city_name):
         "measurement_date": datetime.strptime(
             measurement["date"]["utc"], "%Y-%m-%dT%H:%M:%S%z"
         ),
+        "meta_data": {
+            "entity": measurement["entity"],
+            "sensorType": measurement["sensorType"],
```

Comment:
Can you put a screenshot showing the data in the database after it runs?

Reply:
![image](https://github.com/ECMWFCode4Earth/vAirify/assets/56438486/ccdfd28f-cbab-4360-a2d0-c1c90a7b14b7)